### DISCUSSION TITLE
Wizards

### DISCUSSION THREAD

**Discussion by tyler-hale** [twilio-labs/paste#1352] [Office hours] [answered]
The Consumer Trust and Messaging teams are working together to create an onboarding experience for A2P 10DLC. We are curious which wizard pattern/component we should be using. We have referred to the working draft doc on wizards and would like feedback from Design Systems. 
https://www.figma.com/proto/cuYcSsId5emYE3Bj4hfll1/A2P-Registration?node-id=250%3A41419&scaling=min-zoom&page-id=250%3A40040

**Comment by corinne-nagel** [comment, marked as answer]
Link to Office Hours here: https://www.loom.com/share/45266766429f4455b99e94e6c7af1cca
